components/switch/switch-mixin.js

```diff
@@ -193,7 +196,7 @@ export const SwitchMixin = superclass => class extends FocusMixin(RtlMixin(Focus
 			? this.textPosition : 'end');
 
 		return html`
-			${textPosition === 'start' ? switchLabel : ''}
+			${textPosition === 'start' || textPosition === 'hidden' ? switchLabel : ''}
```

Comment:
Does the switchLabel need to be rendered both before and after the div or would it work to use just one? It would also likely be useful to add a quick comment on why we're rendering it even though there's an `aria-label` on the switch div.

Reply:
Feedback addressed in deaf5d005d77f6505875cbca382f934384fcd016 and 4b9b80f9f12f0527d3dd780250e36ba7db0cb3db